.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 bower_components
-/components
+components
```

Comment:
I think as written this will ignore new files added to some existing directories, such as:

https://github.com/Polymer/docs/tree/master/samples/components

I don't know if there's a good way to write the .gitignore rule so it skips only the versioned component directories, like, `/[0-9.]*/components`, or whether we should rename the existing directories...